docs/other/guides/how-to-create-scaffolding-templates.md

```diff
@@ -0,0 +1,45 @@
+---
+id: creatingScaffoldingTemplates
+title: How to create Scaffolding Templates
+sidebar_label: Creating Scaffolding Templates
+---
+
+### Introduction
+As discussed in [Scaffolding](/docs/get-started/scaffolding), it is possible to save yourself a lot of time by using one of the default Scaffolding Templates.
+
+The great news are that you may also write your own templates! In this article we will go in depth about it.
+
+### Writing your first Scaffolding Template
+We use `node modules` for all of our templates. You should create a new package whose name follows the following pattern: `webiny-cli-scaffold-*`.
+
+For example, valid names could be:
+* webiny-cli-scaffold-append-unicorns
+* webiny-cli-scaffold-add-delicious-donuts-to-my-site
+
+Next, your package must export an array of [Plugins](/docs/deep-dive/plugins-crash-course) (a single Plugin is just as fine). Your Plugins must follow the object structure below:
```

Comment:
`a single Plugin is just as fine` -> `you can also export a single plugin`.

Reply:
done (this falls in the same category as the 1st comment i believe)